cGame_logic.cpp

```diff
@@ -653,11 +654,8 @@ void cGame::shutdown() {
 
     logbook("Allegro MP3 library shut down.");
 
-    // Now we are all neatly closed, we exit Allegro and return to OS.
-    allegro_exit();
-
-    logbook("Allegro shut down.");
-    logbook("Thanks for playing.");
+    // Release the game dev framework, so that it can do cleanup
+    _PLInit.reset();
```

Comment:
I am not 100% sure about how unique ptr works,  but afaik this basically does a `delete` (nothing to replace it with), correct?

I'll look into unique_ptr's more myself in a playground project I have.

Reply:
Yup.
It sets the unique_ptr to nullptr, which deletes the old object it was pointing to.